ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -250,6 +257,15 @@ export default class ConfirmPageContainerContent extends Component {
             }
           />
         )}
+        {showIsSigningOrSubmittingError && (
+          <BannerAlert
```

Comment:
Should we create a special BannerAlert for this error like this, or can we use the existing one from line 223? They seem very similar.

Reply:
I originally did but the one on line 223 is specifically for legacy transactions given the `!supportsEIP1559`.